setup/README.md

```diff
@@ -101,13 +108,18 @@ db.createUser(
 
 
 ### 8. Setup meteor service variables in ~/service/myapp-meteor/run using above ports and mongo credentials
+If you use a custom domain (step 11) you can also use your own domain as `DDP_DEFAULT_CONNECTION_URL` and `ROOT_URL`. Make sure you have a valid HTTPS certificate then. 
+
+It's explained how to get one here for your own domain: https://wiki.uberspace.de/webserver:https .
+
+If you don't want to use HTTPS, just write `https://...`for both URLs. 
 
 ```
 export DISABLE_WEBSOCKETS=1
 #OR
-export DDP_DEFAULT_CONNECTION_URL=http://username.subdomain.uberspace.de:64991/
+export DDP_DEFAULT_CONNECTION_URL=https://username.subdomain.uberspace.de/
```

Comment:
why no port neccessary?

Reply:
I'm honestly not sure, but this was the only way it worked for me. If I added a port, I'd throw an error in my browser that no secure connection could be established.

My best guess would be because the .htaccess automatically redirects to this port 🤷‍♂️

Reply:
Maybe. It's much better without a port anyway. Also because of firewalls.